core/platform/taskqueue/cloud_tasks_emulator_test.py

```diff
@@ -56,19 +64,19 @@ def setUp(self):
             },
             'param3': [1, 2, 3]
         }
-        self.output = []
+        self.output: List[str] = []
         self.unit_test_emulator = cloud_tasks_emulator.Emulator(
             task_handler=self.mock_task_handler, automatic_task_handling=False)
         self.dev_mode_emulator = cloud_tasks_emulator.Emulator(
             task_handler=self.mock_task_handler)
 
-    def test_task_creation_is_handled_correctly(self):
+    def test_task_creation_is_handled_correctly(self) -> None:
         self.assertEqual(self.unit_test_emulator.get_number_of_tasks(), 0)
 
         self.unit_test_emulator.create_task(
-            self.queue_name1, self.url, self.payload1)
+            self.queue_name1, self.url, payload=self.payload1)
```

Comment:
In the emulator, we had a discrepancy. The docstring of `create_task` states that default value is None but this was not the case. So I added a default value `None` to payload in the [create_task](https://github.com/oppia/oppia/blob/2c386ec2edcc8cdf00ace462076cfbea97ddc182/core/platform/taskqueue/cloud_tasks_emulator.py#L178) here.

This can also be seen [here](https://github.com/oppia/oppia/blob/2c386ec2edcc8cdf00ace462076cfbea97ddc182/core/platform/taskqueue/cloud_taskqueue_services.py#L42). A similar function also has a default value None.

Due to this, I had to change payload=payload. 

Please let me know if I'm wrong. Thanks!

Reply:
Okay, no issues then. Thanks for explaining.